front/pages/w/[wId]/assistant/[cId]/index.tsx

```diff
@@ -110,10 +109,9 @@ export default function AssistantConversation({
     const handleRouteChange = () => {
       const assistantSId = router.query.assistantDetails ?? [];
       if (assistantSId && typeof assistantSId === "string") {
-        setIsAssistantDetailOpen(true);
         setDetailViewContent(assistantSId);
       } else {
-        setIsAssistantDetailOpen(false);
+        setDetailViewContent("");
```

Comment:
Shouldn't it be `null`?

Reply:
no setDetailViewContent is a string for the assitant Id (historically doesn't take null, not sure why)

Reply:
do you think we should change it? string | null?

Reply:
"" is falsy 🤦